changelog/8787.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed URL of documentation link in message of warning issued on unregisterd marks.
```

Comment:
```suggestion
Fixed URL of documentation link in message of warning issued on unregistered marks.
```

Reply:
Fixed in https://github.com/pytest-dev/pytest/pull/8830/commits/9afc1af093ddfd7e42c9a0a653bfa0ef8297356f.